hack/mkpr

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+repo="${1}"
+ref="${2}"
+
+if [[ -z "${2}" ]]; then
```

Comment:
so you check if there is a `ref`  (which you call a branch) but not if you set a `repo` ?

Reply:
why use $2 when you set $ref above?